gitpod-io/gitpod
# support git token provider kind in the supervisor

#### What it does

- it should be used as a fallback for now for other IDEs, i.e. Code

#### How to test

- start a workspace: https://ak-git-token-in-supervisor.staging.gitpod-dev.com/#https://github.com/gitpod-io/go-gin-app
- checkout a new branch and try to push from the cli and the SCM view
- switch to Code, test again
- in Code check that links to home, workspaces and access control work properly
- stopping a workspace from Code should work as well
